packages/@aws-cdk/aws-codepipeline-actions/lib/lambda/invoke-action.ts

```diff
@@ -4,6 +4,26 @@ import * as lambda from '@aws-cdk/aws-lambda';
 import { Construct, Stack } from "@aws-cdk/core";
 import { Action } from '../action';
 
+/**
+ * The CodePipeline variables for the Lambda invoke Action.
+ */
+export class LambdaInvokeVariables extends codepipeline.Variables {
+  /**
+   * Reference a CodePipeline variable defined by the Lambda function this action points to.
+   * Variables in Lambda invoke actions are defined by calling the PutJobSuccessResult CodePipeline API call
+   * with the 'outputVariables' property filled.
+   *
+   * @param variableName the name of the variable to reference.
+   *   A variable by this name must be present in the 'outputVariables' section of the PutJobSuccessResult
+   *   request that the Lambda function calls when the action is invoked
+   *
+   * @see https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_PutJobSuccessResult.html
+   */
+  public variable(variableName: string): string {
```

Comment:
Might confuse people why this class has both `variable` and `variableExpression`, no?

Reply:
`variableExpression` is protected, it's not part of the public API of the class, so I don't think that will be an issue.